packages/replay/src/auth.ts

```diff
@@ -1,3 +1,4 @@
+import fetch from "node-fetch";
```

Comment:
This has been "broken"* for months from what I can tell but it seemed like a good idea to fix it now (I run an audit through all of the files in the repo to see if `fetch` is imported by all them). I decided to use `node-fetch` (over `undici`) as that's what is used more by this package (the old CLI)

*didn't always work in node 18 or earlier